### PR TITLE
Fixed background size and changed it to default to opaque color [stag…

### PR DESCRIPTION
**Pull request by rodrigopavezi**
…e-3]

@ghislaineguerin please review on https://editor-stage-3.risevision.com 

@alex-deaconu @ezequielc FYI.

cheers

### CODE REVIEW

**Comment by ghislaineguerin** [comment]
:+1:

File: partials/presentation-properties-modal.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h3 class="modal-title">Presentation Properties</h3>
 
     <div class="form-group">
       <label id="backgroundLabel">Background</label>
-      <background-image-setting id="backgroundImageSettings" background="presentationProperties.background" company-id="{{companyId}}"></background-image-setting>
+      <background-image-setting id="backgroundImageSettings" background="presentationProperties.background" company-id="{{companyId}}" color-parent-container-class="'test'" color-container-class="'test'" position-parent-container-class="'test'" position-container-class="'test'"></background-image-setting>
```
Comment (alex-deaconu) [inline partials/presentation-properties-modal.html]: @rodrigopavezi curious, are those test values in there or that's the actual class name? Thanks.

Comment (rodrigopavezi) [inline partials/presentation-properties-modal.html]: @alex-deaconu it is just a placeholder for @ghislaineguerin actual css classes. cheers

Comment (alex-deaconu) [inline partials/presentation-properties-modal.html]: @rodrigopavezi got it. Thanks for the clarification.

**Pull request merged**